CHANGELOG.md

```diff
@@ -71,83 +71,83 @@ We'd like to offer a big thanks to the 8 contributors who made this release poss
   The `DialogProps` prop has been replaced by a `dialog` component props slot on responsive and mobile pickers:
 
   ```diff
-  // Same on MobileDatePicker, DateTimePicker, MobileDateTimePicker,
-  // TimePicker, MobileTimePicker, DateRangePicker and MobileDateRangePicker.
-  <DatePicker
+   // Same on MobileDatePicker, DateTimePicker, MobileDateTimePicker,
```

Comment:
IMHO—this produces a bit strange result.
Some of our diffs have no preceding spaces and some will do. 🤔 
Maybe we should go with a single approach everywhere?
Or is it some sort of a recommendation I'm not aware of?

Reply:
The change is about always haveing +/-/` ` as a preceding character.